utils/env-file.c

```diff
@@ -0,0 +1,86 @@
+#include "utils/env-file.h"
+
+int create_env_file(void)
+{
+	int fd;
+
+	fd = open(ENV_FILE, O_RDWR | O_CREAT | O_TRUNC, S_IRUSR | S_IWUSR);
+	if (fd < 0)
+		pr_err(ERROR_CREATE_FILE);
+
+	return fd;
+}
+
+
+int open_env_file(void)
+{
+	int fd;
+
+	fd = open(ENV_FILE, O_RDONLY);
+	if (fd < 0)
+		pr_err(ERROR_OPEN_FILE);
+
+	return fd;
+}
+
+/*
+ * write a line to uftrace environment file with given argument fd.
+ * a line contain key-value pair with separator '='.
+ * like this,
+ *
+ * [example]
+ * UFTRACE_PLTHOOK=1
+ */
+void set_env_to_file(int fd, char *key, const char *value)
+{
+	int res;
+	char buf[256];
+
+	sprintf(buf, "%s", key);
+	sprintf(buf + strlen(buf), "%s", "=");
+	sprintf(buf + strlen(buf), "%s\n", value);
```

Comment:
Please do not use `sprintf()` as it's know to vulnerable.  Also why not doing it at once with "%s=%s\n" ?